lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2075,9 +2075,9 @@ def _3d_extend_contour(self, cset, stride=5):
                     continue
 
             stepsize = (len(topverts[0]) - 1) / (nsteps - 1)
-            for i in range(int(round(nsteps)) - 1):
-                i1 = int(round(i * stepsize))
-                i2 = int(round((i + 1) * stepsize))
+            for i in range(nsteps - 1):
```

Comment:
`nsteps` is already a `round`ed integer a few lines earlier. No need to round it again,